### ISSUE TITLE
[homework2]关于Logistic Regression: Training Stability中题目的表达

### ISSUE THREAD

**Issue by Hrc22** [Smadx/USTC-ML24-Fall#53]
可不可分并不影响损失函数有下界0，为什么会影响优化算法的收敛性呢？

(c)中提到Contrast this with the case of non-separable data. Explain why the loss function has a finite lower
 bound in the non-separable case and why gradient descent would converge. 但是对于可分情况也有有限下界🤔

**Comment by Smadx** [comment]
@primehomology

**Comment by WenjinZhang0621** [comment]
这里表述确实不严谨，不过我记得原题大概也如此。确切来说你从(b)就已经可以看出问题所在，这里应该理解为可以在有限点处取得最小值
